stag-library-compiler/src/test/java/com/vimeo/stag/processor/StagProcessorFunctionalTest.java

```diff
@@ -0,0 +1,39 @@
+package com.vimeo.stag.processor;
+
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import javax.annotation.processing.Processor;
+import javax.tools.Diagnostic;
+
+public class StagProcessorFunctionalTest extends AbstractAnnotationProcessorTest {
+
+    private StagProcessor processor;
```

Comment:
This appears unused outside of the `getProcessors` method and can be turned into a local variable.